### DISCUSSION TITLE
element plus自动导入失败

### DISCUSSION THREAD

**Discussion by dddssw** [element-plus/element-plus#13220] [Q&A]
//在vue.config.js,引入官方上的两个插件
const { defineConfig } = require("@vue/cli-service");
module.exports = defineConfig({
  transpileDependencies: true,
  configureWebpack: {
    plugins: [
      require("unplugin-vue-components/webpack")({
        /* options */
      }),
      require("unplugin-auto-import/webpack")({}),
    ],
  },
});
但是最后运行
![image](https://github.com/element-plus/element-plus/assets/58782768/69b5d405-bc8f-42cb-87c5-39f26169b6ab)

**Comment by chenxch** [comment]
这个问题其实你应该要到unplugin-vue-components仓库中提问哦

请参考这个
![image](https://github.com/element-plus/element-plus/assets/23251408/5bb48506-5a51-48c8-9e82-1223d9cffdcb)

**Reply by dddssw** [reply]
感谢您提供的信息,但是我按照上面提供的代码配置vue.config.js文件,还是出现了报错,当前因为插件的问题用不了这个功能吗
![image](https://github.com/element-plus/element-plus/assets/58782768/edc4e86e-6665-45ad-be76-42ea1780580d)

![image](https://github.com/element-plus/element-plus/assets/58782768/fbf02426-c73f-4dd2-a4bf-2c31f314840b)
